packages/okta-react/README.md

```diff
@@ -295,24 +295,30 @@ Retrieves the id token from storage if it exists.
 
 Retrieves the access token from storage if it exists.
 
-#### `auth.login(fromUri)`
+#### `auth.login(fromUri, additionalParams)`
 
-Calls `onAuthRequired` or redirects to Okta if `onAuthRequired` is undefined. This method accepts a `fromUri` parameter to push the user to after successful authentication.
+Calls `onAuthRequired` or redirects to Okta if `onAuthRequired` is undefined. This method accepts a `fromUri` parameter to push the user to after successful authentication, and an optional `additionalParams` object.
+
+For more information on `additionalParams`, see the [`auth.redirect`](#authredirectadditionalparams) method below.
 
 #### `auth.logout(uri)`
 
 Terminates the user's session in Okta and clears all stored tokens. Accepts an optional `uri` parameter to push the user to after logout.
 
-#### `auth.redirect({sessionToken})`
+#### `auth.redirect(additionalParams)`
+
+Performs a full-page redirect to Okta with optional request parameters.
 
-Performs a redirect to Okta with an optional `sessionToken`.
+The `additionalParams` are mapped to the [AuthJS OpenID Connect Options](https://github.com/okta/okta-auth-js#openid-connect-options). This will override any existing [configuration](#configuration-options). As an example, if you have an Okta `sessionToken`, you can bypass the full-page redirect by passing in this token. This is recommended when using the [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget). Simply pass in a `sessionToken` into the `loginRedirect` method follows:
```

Comment:
We should be linking them here instead, because any of these parameters can be used:

https://developer.okta.com/docs/api/resources/oidc#authorize

We should also update the AuthJS readme to link there as well, rather than duplicating the (now stale) options in that readme.